src/RA_AchievementSet.h

```diff
@@ -75,9 +74,9 @@ class AchievementSet
     BOOL HasUnsavedChanges();
 
 private:
-    const AchievementSetType m_nSetType;
+    const AchievementSetType m_nSetType{};
     std::vector<Achievement> m_Achievements;
-    BOOL m_bProcessingActive;
+    BOOL m_bProcessingActive{ TRUE };
```

Comment:
Needed to make sure this class was nothrow default constructible.
The default destructor gives some weird values but I ignored it for now.